src/media/css/forms.styl

```diff
@@ -12,6 +12,7 @@ button, input, select, textarea {
 }
 
 button, input {
+    border-radius: 5px;
```

Comment:
Instead try following 

label, button, input[type=button], input[type=submit] {
    border-radius: 5px;
    cursor: pointer;
}
[data-modal] button{
    border-radius: 0;
}

data-modal will take care of all popup border radius,